File: src/adjustablesmods/ModPlugin.java
package adjustablesmods;

import adjustablesmods.ui.SModRefitButton;
import com.fs.starfarer.api.BaseModPlugin;
import lunalib.lunaRefit.LunaRefitManager;

public class ModPlugin extends BaseModPlugin {
    @Override
    public void onApplicationLoad() {
        LunaRefitManager.addRefitButton(new SModRefitButton());
    }
}

File: src/adjustablesmods/ModPlugin.kt
package adjustablesmods

import adjustablesmods.ui.RefitButton
import com.fs.starfarer.api.BaseModPlugin
import lunalib.lunaRefit.LunaRefitManager

class ModPlugin : BaseModPlugin() {
    override fun onApplicationLoad() {
        LunaRefitManager.addRefitButton(RefitButton())
    }

    override fun onGameLoad(newGame: Boolean) {
        loadData()
        UITabListener.register()
    }
}